NEWS.md

```diff
@@ -1,6 +1,9 @@
 
 # vctrs (development version)
 
+* New `list_sizes()` for computing the size of every element in a list.
+  `list_sizes()` is to `vec_size()` as `lengths()` is to `length()`.
```

Comment:
except that it only supports lists. Atomic vectors and data frames cause an error.